### ISSUE TITLE
"Please relaunch using offical launcher"?

### ISSUE THREAD

**Issue by Wizeguy400** [toontown-strut/toontown-strut#2]
What? is there something i'm missing here?
![image](https://user-images.githubusercontent.com/25600996/164997301-e9550e52-c8b8-42b5-b00d-d51d8cb633c6.png)

**Comment by NodeMixaholic** [comment]
yea this is probably my bad, check astrondb logs if this is your server.

**Comment by NodeMixaholic** [comment]
update: I seem to be having this issue too, patching...

**Comment by NodeMixaholic** [comment]
it seems like it is (for some reason?) not creating users.

**Comment by NodeMixaholic** [comment]
the easiest thing to do would probably be to revert to an older version of TTStrut and put in some new code.

**Comment by NodeMixaholic** [comment]
update 1: hmm older versions seem to be broken too... have you tried installing .NET Framework 3.5?

**Comment by NodeMixaholic** [comment]
even on windows I would have to download .NET 3.5 and reboot to get rid of this, I think I remember that now...

**Comment by NodeMixaholic** [comment]
what platform are you on? I am having a near identical issue on my Mac, I want to see if this is a Mac issue.

**Comment by NodeMixaholic** [comment]
just realised the issue... turns out i forgot a few files

**Comment by NodeMixaholic** [comment]
TLDR: it seems that i forgot that the code doesnt create a default "astrondb" entry and folder, thus I had to fix that by re-adding the one from [toontown stride python3](https://github.com/OSToontown/ToontownStride-Py3)

**Comment by NodeMixaholic** [comment]
Fixed in the latest update.
